css/base/base.css

```diff
@@ -62,6 +62,10 @@ a:not(.toolbar a):focus {
   box-shadow: 0 -2px var(--color-focus), 0 4px var(--color-black);
 }
 
+ul li a {
+  display: block;
+}
```

Comment:
I think this `display: block` is a little too blunt here. It means all links in all lists use `block` which will be problematic if the links are within a string of text.

I think it would be better to set `block` on the specific items where we need it, such as the guides navigation.

Reply:
Yeah I see what you mean, it would definitely cause a problem for strings of text. So this change wouldn't be satisfactory.

The original commit does disregard any other full links within list items throughout the site, which is obviously not as common

Reply:
Hi @nbrettell 

Can you update your PR to remove your `display: block' and then in `guide-nav.css` change this:

```.lgd-guide-nav__list-item > a {
  font-weight: bold;
}
```

to this
```
.lgd-guide-nav__list-item > a {
  display: inline;
  font-weight: bold;
}
```

That seems to fix the issue. If you get that change made, we can merge this then.